core/domain/draft_upgrade_services.py

```diff
@@ -403,10 +363,9 @@ def _convert_states_v33_dict_to_v34_dict(cls, draft_change_list):
             elif (change.property_name ==
                   exp_domain.STATE_PROPERTY_INTERACTION_ANSWER_GROUPS):
                 new_value = [
-                    (state_domain.AnswerGroup.convert_html_in_answer_group(
+                    state_domain.AnswerGroup.convert_html_in_answer_group(
                         answer_group,
-                        conversion_fn,
-                        state_uses_old_rule_spec_schema=True))
+                        conversion_fn)
```

Comment:
Optional: maybe move this to previous line?

Reply:
Done